{"uuid": "f7fbc83a-2da9-4e33-9dfa-478ff96f26bf", "children": ["8aabb36b-e409-4bdf-a386-99e44dfbbd83"], "befores": [{"name": "setup_and_teardown", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: JavaScript code failed\nfrom unknown command: 'Runtime.evaluate' wasn't found\n  (Session info: MicrosoftEdge=140.0.3485.94)\nStacktrace:\n\tGetHandleVerifier [0x0x7ff6a7ba3ec5+18309]\n\t(No symbol) [0x0x7ff6a7b11570]\n\tGetHandleVerifier [0x0x7ff6a7d98efc+2070460]\n\t(No symbol) [0x0x7ff6a79014bd]\n\t(No symbol) [0x0x7ff6a78ffd88]\n\t(No symbol) [0x0x7ff6a790050b]\n\t(No symbol) [0x0x7ff6a7924f22]\n\t(No symbol) [0x0x7ff6a7919c6a]\n\t(No symbol) [0x0x7ff6a7919b45]\n\t(No symbol) [0x0x7ff6a78f39ba]\n\t(No symbol) [0x0x7ff6a78f2391]\n\t(No symbol) [0x0x7ff6a79b3239]\n\t(No symbol) [0x0x7ff6a797fc7a]\n\t(No symbol) [0x0x7ff6a795443b]\n\t(No symbol) [0x0x7ff6a799d082]\n\t(No symbol) [0x0x7ff6a797f943]\n\t(No symbol) [0x0x7ff6a79538e6]\n\t(No symbol) [0x0x7ff6a7952b52]\n\t(No symbol) [0x0x7ff6a7953723]\n\t(No symbol) [0x0x7ff6a7a42f75]\n\t(No symbol) [0x0x7ff6a7a4e74d]\n\tGetHandleVerifier [0x0x7ff6a7c395b3+630387]\n\tGetHandleVerifier [0x0x7ff6a7c41151+662033]\n\t(No symbol) [0x0x7ff6a7b1d659]\n\t(No symbol) [0x0x7ff6a7b16ff4]\n\t(No symbol) [0x0x7ff6a7b17143]\n\t(No symbol) [0x0x7ff6a7b0afc6]\n\tBaseThreadInitThunk [0x0x7ffa7284e8d7+23]\n\tRtlUserThreadStart [0x0x7ffa739c8d9c+44]\n\n", "trace": "  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\tests\\conftest.py\", line 36, in setup_and_teardown\n    driver.maximize_window()\n    ~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 650, in maximize_window\n    self.execute(command, None)\n    ~~~~~~~~~~~~^^^^^^^^^^^^^^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 458, in execute\n    self.error_handler.check_response(response)\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^\n  File \"C:\\Users\\Admin\\PycharmProjects\\PythonHybridFrameworkSelenium\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 232, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1759059992660, "stop": 1759059994845}], "start": 1759059992660, "stop": 1759059995098}